autogen

```diff
@@ -366,27 +382,39 @@ function autogenForFile() {
       # "php script.php" .
       #
       # Note: PHP accepts C, C++, and shell-style comments.
-      echo "<?php"
```

Comment:
I'm not sure if it's valid for a PHP script to have no `<?php` header, will it still parse as a PHP file without it? This is not technically a runline, as there's no `#!/path/to/binary` it's just a marker identifying it as a PHP file.

Reply:
It has been years since I touched PHP... honestly I don't remember. Should I remove it ?

Reply:
I think I should remove it and just leave PHP alone, the NO CODE below also depends on this "> closing comment" so this would break with certain flag combo